.gitignore

```diff
@@ -35,4 +35,7 @@ build
 data
 
 # Ignore logs directory
-logs
\ No newline at end of file
+logs
+
+# Ignore out directory
+out
```

Comment:
where we generate out folder ?

Reply:
This package started generating my IDEA when I switched from the ultimate version to the community.